src/ui/ResultLayoutSelector/ResultLayoutSelector.ts

```diff
@@ -87,6 +89,8 @@ export class ResultLayoutSelector extends Component {
      *
      * By default, the tablet mode breakpoint is at 800 px screen width.
      *
+     * To change this default value, use the {@link SearchInterface.options.responsiveMediumBreakpoint} option.
```

Comment:
(...) use the [`responsiveMediumBreakpoint`]{@link SearchInterface.options.responsiveSmallBreakpoint} option.